### ISSUE TITLE
Flake8 rule 'E901' reported an invalid point

### ISSUE THREAD

**Issue by get-data-** [AtomLinter/linter-flake8#416]
Flake8 reported an invalid point for the rule `E901`, with the messge `SyntaxError: invalid syntax`.

Debug information:
Atom version: 1.16.0
Flake8 version: `2.5.4 (pep8: 1.7.0, mccabe: 0.4.0, pyflakes: 1.0.0) CPython 3.5.2 on Linux`

**Comment by Staberinde** [comment]
Similar issue here:

Flake8 reported an invalid point for the rule `E901`, with the messge `SyntaxError: invalid syntax`.
Requested point: NaN:NaN

flake8 output:
```
tobe@hostname:~/Src/Exercises/Learning/LearningDjango$ flake8 ./teachingGrandmaToSuckEggs/views.py 
./teachingGrandmaToSuckEggs/views.py:8:1: E302 expected 2 blank lines, found 1
./teachingGrandmaToSuckEggs/views.py:12:26: E901 SyntaxError: invalid syntax
./teachingGrandmaToSuckEggs/views.py:14:5: E113 unexpected indentation
tobe@hostname:~/Src/Exercises/Learning/LearningDjango$ 
```

Code:
```
from django.http import HttpResponse
from django.shortcuts import render
from django.views.generic import View

from models import GrandMother
# Create your views here.

def index(request):
    return HttpResponse("GodDamn Django is stupid")


class AViewToAKill(View)

    def get(self, request, *args, **kwargs):
        return HttpResponse("Until we dance into the fire")


class ListGrandMotherView(ListView):

    model = GrandMother
```

Debug information:
Atom version: 1.16.0
Flake8 version: `2.5.4 (pep8: 1.7.0, pyflakes: 1.1.0, mccabe: 0.2.1) CPython 3.5.2 on Linux`

**Comment by Staberinde** [comment]
This issue duplicates https://github.com/AtomLinter/linter-flake8/issues/409

Just trying out the fix mentioned now

**Comment by Arcanemagus** [comment]
@Staberinde exactly correct, there was a bug in how `flake8` interpreted the points for the entire `E9xx` class of errors that was fixed in v3.2.1 (thanks to this check being added and the resulting bug being reported over there!).

**Comment by Arcanemagus** [comment]
Closing due to a lack of response, if you are still having issues after updating `flake8` please provide code to reproduce this!
